operator-common/src/test/java/io/strimzi/operator/common/operator/resource/StrimziPodSetCrdOperatorIT.java

```diff
@@ -215,10 +221,6 @@ public void testUpdateStatusAfterResourceDeletedThrowsKubernetesClientException(
                     return op.reconcile(Reconciliation.DUMMY_RECONCILIATION, namespace, resourceName, null);
                 })
                 .onComplete(context.succeeding(i -> { }))
-                .compose(i -> {
-                    LOGGER.info("Wait for confirmed deletion");
-                    return op.waitFor(Reconciliation.DUMMY_RECONCILIATION, namespace, resourceName, 100L, 10_000L, (n, ns) -> operator().get(namespace, resourceName) == null);
-                })
```

Comment:
With the change in `CrdOperator#internalDelete` to wait for the resource to be gone, this check (and the one in `AbstractCustomResourceOperatorIT`) become redundant. I suppose they were added originally because the `waitFor` in `internalDelete` was not actually waiting for the deletion to be completed.